packages/front-end/components/Tooltip.tsx

```diff
@@ -4,18 +4,22 @@ import { MdInfoOutline } from "react-icons/md";
 // todo: add support for variable positioning instead of just bottom.
 
 interface Props extends HTMLAttributes<HTMLDivElement> {
-  text: string;
+  body: string | JSX.Element;
   tipMinWidth?: string;
   tipPosition?: "bottom" | "top" | "left" | "right";
   innerClassName?: string;
+  textColor?: string;
+  bgColor?: string;
 }
 const Tooltip: FC<Props> = ({
-  text,
+  body,
   children,
   className,
   tipMinWidth = "140px",
   tipPosition = "bottom",
-  innerClassName = "",
+  innerClassName,
+  textColor = "white",
+  bgColor = "dodgerBlue",
```

Comment:
This color isn't part of our current color palette.  The background color for tooltips right now is `#029dd1` which is close, but not exactly the same as dodgerBlue.